src/Debug/WrappedDelayedListener.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace Biig\Component\Domain\Debug;
+
+use Symfony\Component\VarDumper\Caster\ClassStub;
+
+class WrappedDelayedListener
+{
+    /**
+     * @var callable
+     */
+    private $listener;
+
+    /**
+     * @var string
+     */
+    private $pretty;
+
+    private $stub;
+
+    /**
+     * @var mixed|string
+     */
+    private $name;
+
+    private static $hasClassStub;
+
+    /**
+     * WrappedDelayedListener constructor.
+     *
+     * @param $listener
+     */
+    public function __construct($listener)
+    {
+        $this->listener = $listener->getListener();
+
+        $this->name = \is_object($this->listener[0]) ? \get_class($this->listener[0]) : $this->listener[0];
+        $this->pretty = $this->name.'::'.$this->listener[1];
+
+        if (null === self::$hasClassStub) {
+            self::$hasClassStub = class_exists(ClassStub::class);
+        }
+    }
+
+    public function getInfo($eventName)
+    {
+        if (null === $this->stub) {
+            $this->stub = self::$hasClassStub ? new ClassStub($this->pretty.'()', $this->listener) : $this->pretty.'()';
+        }
+
+        return [
+            'event' => $eventName,
+            'priority' => 0,
```

Comment:
Is it normal priority it's always at zero ?

Reply:
Actually no priority is managed by the component. 
I set priority to keep consistency with Event from Symfony.